.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
             name: run tests
             command: |
                 # Poetry isn't installed in the image and need to be installed manually.
-                sudo pip install poetry
+                sudo pip install poetry==1.1.11
```

Comment:
could you add the poetry version as an env on top that we can change easily, so that we dont have to type the version explicitly all the time?

Reply:
Make sense. I applied the requested changes.